node-typescript/README.md

```diff
@@ -56,6 +57,16 @@ We have added Rookout to the original project by:
     const rook = require("rookout/auto_start");
     ```
 
+__IMPORTANT FOR TYPESCRIPT:__
+
+
+* In order to use require() you need to install the node types package to compile:
+    ```bash
+    $ npm install --save-dev @types/node
+    ```
+
+* Rookout need corresponding sourcemaps to work properly.
```

Comment:
the library used (source-map) takes the compiled .js files and checks for sourcemap reference inside it. So even if the mapping is happening inline or in another directory it will work fine as long as sourcemap exist